cypress.config.js

```diff
@@ -0,0 +1,12 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    
+    baseUrl: 'http://automationpractice.com/index.php',
+
+    "env": {
+     "baseUrl":'http://automationpractice.com/index.php'
+  },
+}
+});
```

Comment:
Do not duplicate. Remove one variant. 
` "env": {
     "baseUrl":'http://automationpractice.com/index.php'
  },`